app/routes/press-releases/overview/published.js

```diff
@@ -1,4 +1,21 @@
 import Route from '@ember/routing/route';
+// eslint-disable-next-line ember/no-mixins
+import DataTableRouteMixin from 'ember-data-table/mixins/route';
 
-export default class PressReleasesOverviewPublishedRoute extends Route {
-}
+export default class PressReleasesOverviewPublishedRoute extends Route.extend(DataTableRouteMixin) {
+  modelName = 'press-release';
+
+  mergeQueryOptions(params) {
+    const queryParams = {
+      sort: params.sort,
+      include: 'publication-event',
+      filter: {
+        'publication-event': {
+          ':gt:planned-start-date': new Date().toISOString()
```

Comment:
Voor gepubliceerde persberichten moeten we filteren op `started` ipv op `planned-start-date`.
De filter flag is ook omgekeerd: om datums in het verleden te filteren moet je `:lt` gebruiken.